static/src/javascripts/projects/commercial/modules/third-party-tags/remarketing.js

```diff
@@ -14,9 +10,7 @@ const onLoad = () => {
 };
 
 export const remarketing: () => ThirdPartyTag = () => ({
-    shouldRun:
-        config.get('switches.remarketing') &&
-        !!getAdConsentState(thirdPartyTrackingAdConsent),
+    shouldRun: config.get('switches.remarketing'),
```

Comment:
Shall we add a "default" value of `false` as the 2nd argument in this `config.get`? The code works as is, but the type definition of `shouldRun` in  `ThirdPartyTag` states it returns a boolean, whilst `config.get` returns undefined in some cases where a default value hasn't been specified.

Reply:
👍